### PR TITLE
eslint rules

### PR DESCRIPTION
**Pull request by bennpearson**
### What is the context of this PR?
Added eq-author-eslint-rules npm module to devDependencies 

### How to review 
Check the npm module imports and is hooked up to eslintrc

### CODE REVIEW

**Comment by bennpearson** [comment]
[Standardise eslint rules across apps (1)](https://trello.com/c/5ySzbk5O/117-standardise-eslint-rules-across-apps-1)

File: package.json [hunk 1]
```diff
@@ -50,9 +50,10 @@
     "dotenv": "^4.0.0",
     "enzyme-to-json": "^1.5.1",
     "eslint": "^3.19.0",
+    "eslint-config-author": "https://github.com/ONSdigital/eslint-config-author",
```
Comment (WickyNilliams) [inline package.json]: I wonder if we should make the name more specific? e.g. `eslint-config-ons-author`. Avoids taking a more generic name from the community, and is more obvious what it is

File: .eslintrc [hunk 2]
```diff
@@ -1,73 +1,3 @@
...
-    "react/self-closing-comp": "error",
-
-    "react/jsx-boolean-value": "error",
-    "react/jsx-handler-names": "error",
-    "react/jsx-no-bind": "warn",
-    "react/jsx-pascal-case": "error",
-
-    "babel/no-invalid-this": "error"
-  }
+  "extends": ["eslint-config-author", "eslint-config-author/react"]
```
Comment (WickyNilliams) [inline .eslintrc]: Awesome!

File: package.json (hunk 1 above)
Comment (bennpearson) [inline package.json]: Good point 👍

**Pull request merged**